automator/src/com/testsigma/automator/actions/AddonAction.java

```diff
@@ -389,7 +389,7 @@ private void handleStaleElementExceptionType(Exception e) {
   }
 
   private void handleInvalidStateExceptionType(Exception e) {
-    if (e instanceof ElementNotVisibleException) {
+    if (e instanceof ElementNotInteractableException) {
```

Comment:
https://github.com/SeleniumHQ/selenium/issues/10538 
Since ElementNotInteractableException says about visibility of an DOM element, no changes required